kie-wb-parent/kie-wb-webapp/pom.xml

```diff
@@ -1216,6 +1224,18 @@
       <scope>provided</scope>
     </dependency>
 
+    <dependency>
+      <groupId>org.dashbuilder</groupId>
+      <artifactId>dashbuilder-navigation-client</artifactId>
+      <scope>provided</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>org.dashbuilder</groupId>
+      <artifactId>dashbuilder-cms-client</artifactId>
+      <scope>provided</scope>
+    </dependency>
```

Comment:
Please remove these `provided` client deps. They are not needed. I am preparing a PR which will remove them, but lets just add more.